embedded-graphics/src/fonts/mod.rs

```diff
@@ -11,11 +11,11 @@
 //! The examples below use the [`Font6x8`] font, however any of the [built-in fonts]
 //! in this module or custom fonts can be substituted.
 //!
-//! ## Draw text without using `egtext` macro
+//! ## Print styled "Hello Rust!"
 //!
 //! Text can be drawn to a display by creating a [`Text`] object and attaching a
-//! text style to it by using a [`Styled`] object. By creating the text style manually,
-//! without using the [`egtext`] macro, it can be reused to style multiple text objects.
+//! text style to it by using a [`Styled`] object. This example prints the text
+//! "Hello Rust" with a yellow stroke and blue background.
```

Comment:
IMO texts don't use a stroke, which was the reason to rename `stroke_color` and `fill_color` for `TextStyle`:
```suggestion
//! text style to it by using a [`Styled`] object. This example prints
//! "Hello Rust" with a yellow text on a blue background.
```